Makefile

```diff
@@ -1972,7 +1972,7 @@ JAVA_INCLUDE = -I$(JAVA_HOME)/include/ -I$(JAVA_HOME)/include/linux
 ifeq ($(PLATFORM), OS_SOLARIS)
 	ARCH := $(shell isainfo -b)
 else ifeq ($(PLATFORM), OS_OPENBSD)
-	ifneq (,$(filter amd64 ppc64 ppc64le s390x arm64 aarch64 sparc64, $(MACHINE)))
+	ifneq (,$(filter amd64 ppc64 ppc64le s390x arm64 aarch64 sparc64 loongarch64, $(MACHINE)))
```

Comment:
Is a comparable change also required in CMakeLists.txt?

Reply:
I added -march parameters to cmake_cxx_flags in CMakeLists.txt,there is no problem using the default configuration to build rocksdb